.circleci/Dockerfile

```diff
@@ -1,8 +1,12 @@
-FROM circleci/rust:1.40.0
+# Build with `docker build -t jamwaffles/circleci-embedded-graphics:<rust_version> -f ./.circleci/Dockerfile .`
+# Push with `docker push jamwaffles/circleci-embedded-graphics:<rust_version>`
```

Comment:
You did add docker commands to the `justfile` some time ago, are these the same commands?

Reply:
:facepalm: totally forgot about that, thank you. I've removed this comment and updated the `justfile`.